.devcontainer/cuda11.8-conda/devcontainer.json

```diff
@@ -32,6 +32,7 @@
     "source=${localWorkspaceFolder}/../kvikio,target=/home/coder/kvikio,type=bind,consistency=consistent",
     "source=${localWorkspaceFolder}/../ucxx,target=/home/coder/ucxx,type=bind,consistency=consistent",
     "source=${localWorkspaceFolder}/../cudf,target=/home/coder/cudf,type=bind,consistency=consistent",
+    "source=${localWorkspaceFolder}/../rapidsmpf,target=/home/coder/rapidsmpf,type=bind,consistency=consistent",
```

Comment:
As mentioned in https://github.com/rapidsai/rapidsmpf/pull/238#pullrequestreview-2811764283, [rapidsmpf does _NOT_ support CUDA 11.x](https://github.com/rapidsai/rapidsmpf/pull/144), I think this should be dropped.

Reply:
We don't build CUDA 11 in CI though?

Reply:
Removed the changes here.